### PR TITLE
Disable minio and mysql operators by default and move their crds to their folder

### PR DESCRIPTION
**Pull request by ricosega**
#### Summary
Today I faced that I wanted to deploy the mattermost-operator in our Kubernetes cluster and as of k8s version 1.22 I couldn't.
The cause of this are those crds which are part of minio-operator and mysql-operator found in crds/ folder, because they still use old apiVersion.

So, as they do not have to be applied always and only if the operators are enabled I moved them to their corresponding folder.
Also and no less important, I set the operators as false by default, because they should be deployed separately in a production environment and only enabled for testing purposes as described in the documentation.

### CODE REVIEW

File: charts/mattermost-operator/Chart.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: mattermost-operator
 description: A Helm chart for Mattermost Operator
 type: application
-version: 0.1.1
-appVersion: 1.13.0
+version: 0.1.2
```
Comment (cpanato) [inline charts/mattermost-operator/Chart.yaml:5]: the version should be `0.2.0`

File: charts/mattermost-operator/templates/minio-operator/crd-minioinstances.yaml [hunk 2]
```diff
@@ -36,3 +40,4 @@ spec:
     - name: Replicas
       type: integer
       JSONPath: ".spec.replicas"
+{{- end }}
```
Comment (cpanato) [inline charts/mattermost-operator/templates/minio-operator/crd-minioinstances.yaml:43]: nit: newline

File: charts/mattermost-operator/templates/mysql-operator/crd-mysqlbackups.yaml [hunk 3]
```diff
@@ -8,10 +8,15 @@ metadata:
...
+    "helm.sh/hook-delete-policy" : post-delete 
+    "helm.sh/hook-weight": "-4"  
 spec:
   group: mysql.presslabs.org
   names:
     kind: MysqlBackup
     plural: mysqlbackups
   scope: Namespaced
   version: v1alpha1
+{{- end }}
```
Comment (cpanato) [inline charts/mattermost-operator/templates/mysql-operator/crd-mysqlbackups.yaml:22]: nit: newline

File: charts/mattermost-operator/templates/mysql-operator/crd-mysqlclusters.yaml [hunk 4]
```diff
@@ -34,3 +38,4 @@ spec:
       statusReplicasPath: .status.readyNodes
     status: {}
   version: v1alpha1
+{{- end }}
```
Comment (cpanato) [inline charts/mattermost-operator/templates/mysql-operator/crd-mysqlclusters.yaml:41]: nit: newline

**Comment by ricosega** [comment]
Hi @cpanato comments fixed now.

**Comment by ricosega** [comment]
@Szymongib @stylianosrigas can someone check this?

**Review by stylianosrigas — APPROVED** [review]
LGTM @ricosega thanks a lot. We need to update this to the 1.17.0 Operator version but this needs some additional changes to the templates, so we can do it in a separate PR.

**Review by Szymongib — APPROVED** [review]
No blockers from my side, I will defer to Stylianos judgment on this.

**Review by cpanato — APPROVED** [review]

**Pull request merged**